doc/concepts/morphology.rst

```diff
@@ -144,11 +206,24 @@ uses 4 segments to model the soma.
 Morphology
 ----------
 
-A *morphology* describes the geometry of a cell as unbranched cables with variable radius,
-and their associated tree structure.
-Every segment tree can be used to generate a unique morphology, which derives and enumerates
-*branches* from the segments.
-The branches of a morphology are unbranched cables, composed of one or more segments, where:
+.. glossary::
+
+  morphology
+    Morphologies in Arbor are modelled as a set of one dimensional :term:`cables <cable>` of variable radius,
+    joined together to form a tree. Only :ref:`cable cells <modelcablecell>` support custom
+    morphologies in Arbor. Morphologies can be create by :ref:`loading a file<morph-formats>` with a cell description,
+    or by manually constructing one from a :term:`segment tree`.
+
+A segment tree and a morphology can both describe the exact same cable cell geometry,
```

Comment:
I find this sentence misleading: "A segment tree and a morphology can both describe the exact same cable cell geometry". 
Segment trees by themselves can not describe the cell geometry. They have to be used to create morphologies. As such I don;t think we can compare the two. Maybe what you mean to say is: "A segment tree differs from a morphology mainly in it's coordinate system ... "

Reply:
1. Why can't a segment tree describe a cell geometry?
2. You can instantiate a morphology from just a segment tree.
3. The difference are two-fold: the coordinate system and the fact that angles between branches are not defined in morphologies, or, equivalently, the branching points have no specific position in space, other than in relation to a specific distance from other branching points through any branch that links them. I've added that, it is more complete.

Reply:
1. Because you can't build a cable cell from a segment tree, you need to use a morphology. 
The morphology **has to** describe the same geometry as the segment tree it was built from. So saying that a segment tree and a morphology *can* describe the same geometry sounds strange: either the morphology was built from that segment tree and therefore they **have to** describe the same thing; or the morphology was built from a different segment tree in which case it makes more sense to compare the underlying segment trees or the resulting morphologies.

Reply:
Actually, you can instantiate a `cable_cell` with either a `morphology` and a `segment_tree`, therefore they must (be able to) describe the same cell geometry, right?

Of course a morphology created from a segment tree will describe the same thing. Maybe the word 'can' is the source of conflict, I use it because not every segment tree and morphology describe the same geometry. I'll add a second sentence stating this.

Reply:
> Actually, you can instantiate a cable_cell with either a morphology and a segment_tree, therefore they must (be able to) describe the same cell geometry, right?

True, but the `segment_tree` is implicitly used to construct a morphology. 

Yes, I think the "can" is the issue here.